### PR TITLE
Pure CSS reorganize

### PR DESCRIPTION
**Pull request by jonahdlin**
CSS reorganization. This one should only have CSS changes.

### CODE REVIEW

**Comment by jonahdlin** [comment]
pathways.txt should be deleted anyway, since it was mistakenly pushed in the first place

File: src/styles/features/view/menuBar.css [hunk 1]
```diff
@@ -33,6 +36,7 @@
 .title-container {
   padding-left: calc(1em - var(--icon-size) / 3);
   padding-right: 1em;
+
   color: var(--midnight-blue);
```
Comment (maxkfranz) [inline src/styles/features/view/menuBar.css]: Var names should be semantic, not the name of the colour.  What if in future you want to change the variable value and it's not blue?

Comment (jonahdlin) [reply src/styles/features/view/menuBar.css]: Names updated. They no longer use the name of the colour, instead using a descriptor of where the colour should be used.

**Comment by jonahdlin** [comment]
Redone in #108

**Pull request closed without merging**